Client/README.md

```
@@ -0,0 +1,46 @@
+# Auto.LLM Client
+
+This is the client application for Auto.LLM, which allows an LLM to control a computer.
+
+## Requirements
+
+### Platform Support
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)